docs/metadata.md

```diff
@@ -20,19 +20,37 @@ type Arbitrable @entity {
 }
 ```
 
-In the frontend we try to parse the `metadata` as a json object.
+In the frontend we try to parse the `metadata` as a JSON object.
 
-### metadata - Json object
+### metadata - JSON object
 
 We are going to try to find the following keys:
 
 1. `description`
 2. `metadata`
 
+If `metadata` is a valid IPFS hash/cid we'll fetch its content.
```

Comment:
```suggestion
If `metadata` is a valid [native IPFS path](https://docs.ipfs.io/guides/guides/addressing/#native-urls) we'll fetch its content through IPFS.
```

Wanted to double check, we are assuming the `ipfs://` protocol prefix, right (although we still attempt to fallback to detecting if it's a CID/IPLD path if it doesn't exist)?

Reply:
Oh yeah good catch.

Reply:
We expect to be in the form of `ipfs:{cid | hash}`. Not sure if it's the same?

Reply:
Ah shit, no it isn't :( (it needs the `://`, not just the `:`).

Nevermind, I've changed the suggestion to something else!